ruby/vampires.rb

```diff
@@ -0,0 +1,37 @@
+
+puts "How many employees are being processed?"
+number_of_employees_to_process = gets.chomp
+until number_of_employees_to_process == 0
+	puts "What is your name?"
+	employee_name = gets.chomp
+	puts "How old are you?" 
```

Comment:
You can change an input variable's class by chaining to the `to_i` method on to `gets.chomp` so `gets.chomp.to_i` instead of continually changing it further on in your code.